packages/amplify_authenticator/lib/src/enums/username_input.dart

```diff
@@ -13,17 +13,14 @@
  * permissions and limitations under the License.
  */
 
-enum UsernameAttribute { email, phoneNumber }
+enum UsernameType { username, email, phoneNumber }
```

Comment:
We named this UsernameAttribute and exclused 'username' to tie it to the Cognito configuration, which is often confusing. I thought that creating any abstraction around this might add to confusion down the road. But I agree that what you have here is cleaner code and makes more sense within the Authenticator. Could you maybe just add some documentation here to reference the cognito usernameAttribute value?